dbms/src/Common/TiFlashMetrics.h

```diff
@@ -150,6 +150,8 @@ namespace DB
         F(type_exec, {{"type", "exec"}}, ExpBuckets{0.0005, 2, 20}),                                                                      \
         F(type_migrate, {{"type", "migrate"}}, ExpBuckets{0.0005, 2, 20}),                                                                \
         F(type_v3, {{"type", "v3"}}, ExpBuckets{0.0005, 2, 20}))                                                                          \
+    M(tiflash_storage_page_write_batch_size, "The size of each write batch in bytes", Histogram,                                          \
+        F(type_v3, {{"type", "v3"}}, EqualWidthBuckets{1 * 1024 * 1024, 100, 10 * 1024 * 1024}))                                          \
```

Comment:
... and 100 buckets with each 10MiB is too large for the IO size to PageStorage

Reply:
But the default large write threshold is 250MB. So we change it to 30 buckets with each 10MB?

Reply:
maybe we can use `ExpBuckets` begin with 4kb and base=4 and size=10?

> In [16]: ', '.join(['{}KiB'.format(4 * 4**i) if 4 * 4**i < 1024.0 else '{}MiB'.format(4 * 4**i / 1024.0) for i in xrange(10)])
> Out[16]: '4KiB, 16KiB, 64KiB, 256KiB, 1.0MiB, 4.0MiB, 16.0MiB, 64.0MiB, 256.0MiB, 1024.0MiB'

Reply:
Fixed.